07_modules_libs/homework/server/server.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/rabbit72/golang-training-2021/07_modules_libs/homework/stock"
+)
+
+func getTickerStat(w http.ResponseWriter, r *http.Request) {
+	ticker := chi.URLParam(r, "ticker")
+
+	stockStat, err := stock.GetStockStat(ticker, apiKey)
+	if err != nil {
+		log.Printf("Cannot get stock data for %v - %v", ticker, err)
+		http.Error(w, fmt.Errorf("the ticker %v has not been found", ticker).Error(), 404)
+		return
+	}
+	payload, err := json.Marshal(stockStat)
+	if err != nil {
```

Comment:
This `if-else` statement is fine, but Go way code prefer using return and avoid using else flow branch